src/AppInstallerCLICore/Commands/RootCommand.cpp

```diff
@@ -110,6 +110,24 @@ namespace AppInstaller::CLI
                 }
             }
         }
+
+        void OutputAdminSettings(Execution::Context& context)
+        {
+            auto info = context.Reporter.Info();
+            info << std::endl;
+
+            Execution::TableOutput<2> adminSettingsTable{ context.Reporter, { Resource::String::AdminSettingHeader, Resource::String::PoliciesState } };

```

Comment:
We should probably rename PoliciesState to a generic name Or create a new string for AdminSettingState

Same for PoliciesEnabled and PoliciesDisabled below